### PR TITLE
alignments v1 

### PR DESCRIPTION
**Pull request by eitanz-coti**
alignments v1

### CODE REVIEW

File: src/account/confidential-account.ts [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
-import { onboard } from "./onboard"
+import {BaseWallet, Contract, Provider, Wallet} from "ethers"
+import {buildInputText} from "../crypto_utils"
+import {onboard} from "./onboard"
+import {decryptUint, initEtherProvider} from "../utils";
 
 export class ConfidentialAccount {
-  constructor(readonly wallet: BaseWallet, readonly userKey: string) {}
+    constructor(readonly wallet: BaseWallet, readonly userKey: string) {
+        console.log("eitan ConfidentialAccount")
```
Comment (gmesika-coti) [inline src/account/confidential-account.ts]: ?

File: src/libs/libs_readme.md [hunk 2]
```diff
@@ -1,66 +1,86 @@
-# crypto.ts
+# crypto_utils.ts
```
Comment (gmesika-coti) [inline src/libs/libs_readme.md:1]: why to have only readme in the dir?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,14 @@
 # COTI V2 Confidentiality Preserving L2 | SDKs and Examples
 
-All repositories specified below contain smart contracts that implement confidentiality features using the COTI V2 protocol.
-The contracts provide examples for various use cases, such as Non-Fungible Tokens (NFTs), ERC20 tokens, Auction, and Identity management.
+All repositories specified below contain smart contracts that implement confidentiality features using the COTI V2
```
Comment (gmesika-coti) [inline README.md:3]: please make sure no changes of spaces or new lines for no reason

File: README.md [hunk 4]
```diff
@@ -18,8 +22,8 @@ Interact with the network using any of the following:
 
 The following contracts are available in each of the packages:
 
-| Contract                       |            | python sdk  | hardhat sdk | typescript sdk | Contract Description                                                                                                                          |
-|--------------------------------|------------|-------------|-------------|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| Contract                       |            | python sdk | hardhat sdk | typescript sdk | Contract Description                                                                                                                          |
+|--------------------------------|------------|------------|-------------|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (gmesika-coti) [inline README.md:26]: messed table

File: src/utils.ts [hunk 5]
```diff
@@ -0,0 +1,155 @@
+import {ethers, JsonRpcProvider, Provider, toNumber, TransactionRequest, Wallet,} from "ethers";
```
Comment (gmesika-coti) [inline src/utils.ts]: this files should have being ethers-utils

**Review by gmesika-coti — CHANGES_REQUESTED** [review]

File: src/account/confidential-account.ts (hunk 1 above)
Comment (eitanz-coti) [reply src/account/confidential-account.ts]: removed.

File: src/utils.ts (hunk 5 above)
Comment (eitanz-coti) [reply src/utils.ts]: renamed

File: README.md (hunk 3 above)
Comment (eitanz-coti) [reply README.md:3]: no changes appears in github.

**Comment by eitanz-coti** [comment]
merged already.

**Pull request closed without merging**